app/resources/documentation/manual/index.md

```diff
@@ -939,15 +939,15 @@ The entity property editor is split into two separate areas. At the top, there i
 
 ### Editing Properties
 
-To select an entity property, just click in the row that represents that property in the table. The clicked field will be highlighted, indicating that it has focus. The highlight indicates that you can change the field by entering text. In the screenshot above, the "mangle" property has been selected, and its value has focus, indicating that it is ready to be changed.
+To select an entity property, just click in the row that represents that property in the table. The row will be highlighted, and at the same time, the field which you clicked on will get a black outline. The black outline is called a _cursor_ and it indicates that you can change the outlined field by entering text. In the screenshot above, the "origin" property has been selected, and its key has the cursor, indicating that it is ready to be changed.
```

Comment:
Something is wrong here, I think all of these are undoing correct changes. E.g. mangle is selected, not origin

Reply:
And I'm pretty sure all of the OS caption changes are reverts as well.

Reply:
You are right. I'll revert all of that.